docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+version: '3.8'
+services:
+
+  postgresql-db:
+    image: postgres:14.7-alpine
+    user: postgres
+    environment:
+      POSTGRES_DB: "employeesdepartments"
+      POSTGRES_PASSWORD: "postgres"
```

Comment:
Can you please make it and environment variable and don't hardcode

Reply:
Added .env file with environment variables.